### PR TITLE
docs(configuration) : Remove hashed and total-size for optimization.moduleIds

### PR DESCRIPTION
**Pull request by jeffin143**
Handling : #2601

### CODE REVIEW

File: src/content/configuration/optimization.md [hunk 1]
```diff
@@ -238,6 +236,7 @@ module.exports = {
   ]
 };
 ```
+W> `moduleIds: 'hashed'` is deprecated in favour of `deteministic` and also `moduleIds: total-size` has been remove in V5.
```
Comment (montogeek) [inline src/content/configuration/optimization.md]:
```suggestion
W> `moduleIds: 'hashed'` is deprecated in favour of `deterministic` and also `moduleIds: total-size` has been remove in V5.
```

Comment (jeffin143) [reply src/content/configuration/optimization.md]: Done, also solved lint issue :)

**Comment by jeffin143** [comment]
@montogeek friendly ping :)

**Comment by montogeek** [comment]
Thanks!

File: src/content/configuration/optimization.md [hunk 2]
```diff
@@ -239,6 +237,8 @@ module.exports = {
 };
 ```
 
+W> `moduleIds: 'hashed'` is deprecated in favour of `deterministic` and also `moduleIds: total-size` has been remove in V5.
```
Comment (EugeneHlushko) [inline src/content/configuration/optimization.md]: This is not complete, i will follow as it was already merged @montogeek

**Pull request merged**